### PR TITLE
Add support for am62 kirkstone

### PR DESCRIPTION
**Pull request by RikardO-HM**

### CODE REVIEW

File: kirkstone-next.xml [hunk 1]
```diff
@@ -40,4 +42,10 @@
   </project>
   <project remote="freescale" upstream="kirkstone"   name="meta-freescale-ml"       path="sources/meta-freescale-ml"/>
   <project remote="freescale" upstream="kirkstone"   name="meta-freescale-3rdparty" path="sources/meta-freescale-3rdparty"/>
+
+  <project remote="yocto"  upstream="kirkstone" name="meta-arm" path="sources/meta-arm"/>
+  <project  remote="yocto"  upstream="kirkstone" name="meta-ti" path="sources/meta-ti"/>
+
+  <project  remote="tdx" upstream="kirkstone-6.x.y" name="meta-toradex-bsp-common.git" path="sources/meta-toradex-bsp-common"/>
+  <project  remote="tdx" upstream="kirkstone-6.x.y" name="meta-toradex-ti.git" path="sources/meta-toradex-ti"/>
```
Comment (albsod) [inline kirkstone-next.xml:50]:
```suggestion
  <project remote="yocto"  upstream="kirkstone" name="meta-ti" path="sources/meta-ti"/>

  <project remote="tdx" upstream="kirkstone-6.x.y" name="meta-toradex-bsp-common.git" path="sources/meta-toradex-bsp-common"/>
  <project remote="tdx" upstream="kirkstone-6.x.y" name="meta-toradex-ti.git" path="sources/meta-toradex-ti"/>
```

File: kirkstone.xml [hunk 2]
```diff
@@ -40,6 +42,9 @@
   </project>
   <project revision="1c107bc6dc6437be87fbcd6a91528dba316384cf" remote="freescale" upstream="kirkstone"   name="meta-freescale-ml"       path="sources/meta-freescale-ml"/>
   <project revision="9e94b64bdfebcf7bfdf2af6447cec866a4efa814" remote="freescale" upstream="kirkstone"   name="meta-freescale-3rdparty" path="sources/meta-freescale-3rdparty"/>
-</manifest>
-
+  <project revision="260e3adc2bf322f52d81c0642c825088a88bb051" remote="yocto"  upstream="kirkstone" name="meta-arm" path="sources/meta-arm"/>
+  <project revision="c57b63147d2ac0dde16e43f7407d0ff50e62efdb" remote="yocto"  upstream="kirkstone" name="meta-ti" path="sources/meta-ti"/>
 
+  <project revision="0cb7188573584dd9db39e5689e843edbe9440588" remote="tdx" upstream="kirkstone-6.x.y" name="meta-toradex-bsp-common.git" path="sources/meta-toradex-bsp-common"/>
+  <project  revision="kirkstone-6.x.y"  remote="tdx" upstream="kirkstone-6.x.y" name="meta-toradex-ti.git" path="sources/meta-toradex-ti"/>
```
Comment (albsod) [inline kirkstone.xml:49]:
```suggestion
  <project revision="kirkstone-6.x.y" remote="tdx" upstream="kirkstone-6.x.y" name="meta-toradex-ti.git" path="sources/meta-toradex-ti"/>
```

File: kirkstone.xml [hunk 3]
```diff
@@ -40,6 +42,9 @@
   </project>
   <project revision="1c107bc6dc6437be87fbcd6a91528dba316384cf" remote="freescale" upstream="kirkstone"   name="meta-freescale-ml"       path="sources/meta-freescale-ml"/>
   <project revision="9e94b64bdfebcf7bfdf2af6447cec866a4efa814" remote="freescale" upstream="kirkstone"   name="meta-freescale-3rdparty" path="sources/meta-freescale-3rdparty"/>
-</manifest>
-
+  <project revision="260e3adc2bf322f52d81c0642c825088a88bb051" remote="yocto"  upstream="kirkstone" name="meta-arm" path="sources/meta-arm"/>
+  <project revision="c57b63147d2ac0dde16e43f7407d0ff50e62efdb" remote="yocto"  upstream="kirkstone" name="meta-ti" path="sources/meta-ti"/>
```
Comment (albsod) [inline kirkstone.xml:46]:
```suggestion
  <project revision="260e3adc2bf322f52d81c0642c825088a88bb051" remote="yocto" upstream="kirkstone" name="meta-arm" path="sources/meta-arm"/>
  <project revision="c57b63147d2ac0dde16e43f7407d0ff50e62efdb" remote="yocto" upstream="kirkstone" name="meta-ti" path="sources/meta-ti"/>
```

File: scripts/bygg [hunk 4]
```diff
@@ -124,10 +124,21 @@ parse_options()
...
+        imx8mp-var-dart-hmx1)
+            machine_folder="hmx"
+            ;;
+        verdin-am62-hmm)
+            machine_folder="hmm"
+            ;;
+        *)
+            machine_folder="generic"
+            ;;
+    esac
```
Comment (albsod) [inline scripts/bygg:141]: Use 2-space indentation as rest of the  script please

File: scripts/bygg [hunk 5]
```diff
@@ -169,9 +180,19 @@ get_build_folder()
...
+            ;;
+        imx8mp-var-dart-hmx1)
+            echo "build-hmx"
+            ;;
+        verdin-am62-hmm)
+            echo "build-hmm"
+            ;;
+        *)
+            echo "build"
+            ;;
```
Comment (albsod) [inline scripts/bygg:195]: 2-space indentation

**Review by albsod — COMMENTED** [review]
Great! Just some white-space nitpicking

File: kirkstone.xml (hunk 2 above)
Comment (albsod) [reply kirkstone.xml:49]: Hmm, shouldn't it be kirkstone-6.x.y (floating) only on kirkstone-next and a specified version on kirkstone?

Comment (RikardO-HM) [reply kirkstone.xml]: correct, now fixed

File: scripts/bygg (hunk 4 above)
Comment (RikardO-HM) [reply scripts/bygg]: fixed

File: scripts/bygg (hunk 5 above)
Comment (RikardO-HM) [reply scripts/bygg]: fixed

File: kirkstone-next.xml [hunk 6]
```diff
@@ -38,6 +40,12 @@
   <project remote="variscite" upstream="kirkstone"   name="meta-variscite-sdk"      path="sources/meta-variscite-sdk">
       <linkfile src="scripts/var-setup-release.sh" dest="var-setup-release.sh"/>
   </project>
-  <project remote="freescale" upstream="kirkstone"   name="meta-freescale-ml"       path="sources/meta-freescale-ml"/>
-  <project remote="freescale" upstream="kirkstone"   name="meta-freescale-3rdparty" path="sources/meta-freescale-3rdparty"/>
+  <project remote="freescale" upstream="kirkstone" name="meta-freescale-ml"       path="sources/meta-freescale-ml"/>
```
Comment (albsod) [inline kirkstone-next.xml:43]:
```suggestion
  <project remote="freescale" upstream="kirkstone" name="meta-freescale-ml" path="sources/meta-freescale-ml"/>
```

**Review by albsod — APPROVED** [review]

**Pull request merged**